docs/en/interfaces/formats.md

```diff
@@ -711,6 +712,50 @@ ClickHouse inputs and outputs protobuf messages in the `length-delimited` format
 It means before every message should be written its length as a [varint](https://developers.google.com/protocol-buffers/docs/encoding#varints).
 See also [how to read/write length-delimited protobuf messages in popular languages](https://cwiki.apache.org/confluence/display/GEODE/Delimiting+Protobuf+Messages).
 
+## Parquet {#data-format-parquet}
+
+[Apache Parquet](http://parquet.apache.org/) format. ClickHouse supports read and write operations for this format.
+
+### Data Types Matching
+
+Some of the Parquet data types are not supported. The table below shows the supported data types and how they match ClickHouse [data types](../data_types/index.md) in `INSERT` and `SELECT` queries.
```

Comment:
Remove first sentence + add unsupported types to table below (or as separate list)